tests/puterbot/test_scrub.py

```diff
@@ -0,0 +1,225 @@
+"""Module to test scrub.py"""
+from puterbot.scrub import scrub
+
+
+def test_empty_string() -> None:
+    """ This test is to ensure that an empty string is returned 
+    if an empty string is passed to the scrub function.
+    
+    Args:
+        None
+    Returns:
+        None
+    Raises: 
+        None
+    """
```

Comment:
No need to document like this for tests 👍

Reply:
noted!